bigtable/client/client_options.h

```diff
@@ -53,6 +53,36 @@ class ClientOptions {
     return *this;
   }
 
+  void SetCompressionAlgorithm(grpc::ChannelArguments *channel_arguments,
+                               const grpc_compression_algorithm algorithm);
+
+  void SetGrpclbFallbackTimeout(grpc::ChannelArguments *channel_arguments,
```

Comment:
Why do you need to pass a `channel_arguments` parameter?  The class already has a `channel_arguments_` member, that is the one we want to modify, right?

Reply:
Done.